vhw.go

```diff
@@ -0,0 +1,28 @@
+package nmea
+
+const (
+	// TypeVHW type for VHW sentences
+	TypeVHW = "VHW"
+)
+
+// VHW contains information about water speed and heading
+type VHW struct {
+	BaseSentence
+	TrueHeading            float64
+	MagneticHeading        float64
+	SpeedThroughWaterKnots float64
+	SpeedThroughWaterKPH   float64
```

Comment:
The fact that it's going through water is implied by the context. I think `SpeedKnots` and `SpeedKPH` would be better.

Reply:
The reason for the naming is to avoid confusion with ‘GroundSpeedKnots’ in https://github.com/adrianmo/go-nmea/blob/6384a696ae32a6e2ed0fd9374309a579b5b98603/vtg.go#L14. But I’m fine with changing it to ‘SpeedKnots’ and ‘SpeedKPH’ if the context implies it. 👍

Reply:
you're right